src/EditorFeatures/CSharp/Outlining/Outliners/DocumentationCommentOutliner.cs

```diff
@@ -50,24 +51,26 @@ private static string GetBannerText(DocumentationCommentTriviaSyntax documentati
                     else if (node is XmlEmptyElementSyntax)
                     {
                         var e = (XmlEmptyElementSyntax)node;
-                        var cref = e.Attributes.OfType<XmlCrefAttributeSyntax>().FirstOrDefault();
-                        if (cref != null)
+                        foreach (var attribute in e.Attributes)
                         {
-                            sb.Append(" ");
-                            sb.Append(cref.Cref.ToString());
-                        }
-
-                        var nameattribute = e.Attributes.OfType<XmlNameAttributeSyntax>().FirstOrDefault();
-                        if (nameattribute != null)
-                        {
-                            sb.Append(" ");
-                            sb.Append(nameattribute.Identifier.Identifier.Text);
-                        }
-
-                        var langword = e.Attributes.OfType<XmlTextAttributeSyntax>().FirstOrDefault(a => a.Name.LocalName.ToString() == "langword");
-                        if (langword != null)
-                        {
-                            AppendTextTokens(sb, langword.TextTokens);
+                            if (attribute is XmlCrefAttributeSyntax)
+                            {
+                                sb.Append(" ");
+                                sb.Append(((XmlCrefAttributeSyntax)attribute).Cref.ToString());
+                            }
+                            else if (attribute is XmlNameAttributeSyntax)
+                            {
+                                sb.Append(" ");
+                                sb.Append(((XmlNameAttributeSyntax)attribute).Identifier.Identifier.Text);
+                            }
+                            else if (attribute is XmlTextAttributeSyntax)
+                            {
+                                AppendTextTokens(sb, ((XmlTextAttributeSyntax)attribute).TextTokens);
+                            }
+                            else
+                            {
+                                Debug.Fail($"Unexpected XML syntax kind {attribute.Kind()}");
```

Comment:
The base type is `XmlAttributeSyntax` and it doesn't directly expose the value of the attribute, so I can't get anything to display.  This case is unlikely to hit, though, because the only three types that inherit from `XmlAttributeSyntax` are covered here; this is more of a sanity-check for if another type is added.